input/pagecontent/conformance.md

```diff
@@ -29,42 +29,77 @@ Supporting a profile requires implementation of certain behaviors. In particular
 
 These are requirements in the base FHIR specification, not additional requirements imposed by mCODE. Refer to the [FHIR Documentation on supported profiles](https://www.hl7.org/fhir/profiling.html#CapabilityStatement.rest.resource.supportedProfile) for details.
 
+#### Supported Operations
 
-##### Supported Operations
+##### mCODE Data Sender
 
-TBD
+1. **List mCODE Patients** (defined below). mCODE Data Senders SHALL implement AT LEAST ONE of the following operations UNLESS they are a specialty system that does not implement `CancerPatient` due to unavailable data as described above.
 
-#### mCODE Patients
+  a. For participants where ALL patients with confirmed cancer diagnoses are covered by mCODE:
+
+      GET [base]/Condition?code:in=http://hl7.org/fhir/us/mcode/ValueSet/mcode-primary-or-uncertain-behavior-cancer-disorder-vs
+
+    Given the results of this query, `Patient` resources can be retrieved by iterating through the returned Conditions client-side and retrieving Patients from the standard `GET [base]/Patient/:id` endpoint.
+
+    Additionally, participants MAY support `_include` to get the relevant `Patient` resources in a single request:
+
+        GET [base]/Condition?code:in=http://hl7.org/fhir/us/mcode/ValueSet/mcode-primary-or-uncertain-behavior-cancer-disorder-vs&_include=Observation:patient
+
+    Additionally, participants MAY support reverse chaining to get _only_ the `Patient` resources in a single request:
+
+        # TODO not sure if this is actually possible -- remove if it's not.
+
+        GET [base]/Patient?_has:Observation:patient:code:in=http://hl7.org/fhir/us/mcode/ValueSet/mcode-primary-or-uncertain-behavior-cancer-disorder-vs&_include=Observation:patient
```

Comment:
This part: `_has:Observation:patient:code` doesn't seem right. It is a Condition that has a code.

Reply:
I've fixed this and the other API request parameter issue you pointed out above.

It's not possible to test `code:in=` with Logica because there are more than 1000 codes in the value set, but I have tested now with a fixed `code=` now.